lib/smooth/application/network/http/regular/MIMEParser.cpp

```diff
@@ -82,6 +84,42 @@ namespace smooth::application::network::http::regular
         return mode != Mode::None;
     }
 
+    auto MIMEParser::find_boundary() const
+    {
+        BoundaryIterator p = data.cend();
+
+        if (data.size() > bound.size() + 2)
```

Comment:
Can't I believe I left magic numbers in my code.....

Lets replace `2` with a constant `LEN_OF_CRLF`

Reply:
o.k. fixed